renderer/src/components/WalletModule.tsx

```diff
@@ -68,12 +69,24 @@ const WalletModule: FC<PropsWallet> = ({ isOpen = false }) => {
           {stationAddress.slice(stationAddress.length - 6, stationAddress.length)}
         </p>
         <button
-          className={'h-full w-auto flex flex-row items-center justify-start group ml-2 cursor-pointer'}
+          className={'h-full w-auto flex flex-row items-center justify-start group ml-2 mr-2 cursor-pointer'}
           tabIndex={0}
           onClick={() => navigator.clipboard.writeText(stationAddress) }
         >
           <CopyIcon className="btn-icon-primary mr-1 h-[12px] opacity-80 group-hover:opacity-100" />
-          <span className='text text-body-3xs text-white hidden group-hover:block opacity-80'>Copy</span>
+        </button>
+        <p className='station-address text text-body-3xs text-white'>
+          {' '}
+          {stationAddress0x.slice(0, 6)}
+          {' '}. . .{' '}
+          {stationAddress0x.slice(stationAddress0x.length - 6, stationAddress0x.length)}
+        </p>
+        <button
+          className={'h-full w-auto flex flex-row items-center justify-start group ml-2 cursor-pointer'}
+          tabIndex={1}
+          onClick={() => navigator.clipboard.writeText(stationAddress0x) }
+        >
+          <CopyIcon className="btn-icon-primary mr-1 h-[12px] opacity-80 group-hover:opacity-100" />
```

Comment:
We need to display 2 addresses now, for compatibility with different wallets:

<img width="464" alt="Screenshot 2023-10-24 at 15 03 16" src="https://github.com/filecoin-station/desktop/assets/10247/47b197e9-eb2c-4023-9153-d980e0758faf">

Reply:
I wonder if we need a ? icon that opens a modal describing why there are two addresses?

Reply:
This would be an improvement for sure. I don't think however we have a UI element for this :| Would you consider this a release blocker? I think we can get away without

Reply:
Seeing that you approved the PR, I think this isn't a release blocker